app/src/main/res/values-et/strings.xml

```diff
@@ -73,8 +73,8 @@
     <string name="content_description_move_to_archive">Teisalda arhiivi</string>
     <string name="content_description_block_the_user">Blokeeri</string>
     <string name="content_description_unblock_the_user">Tühista blokeering</string>
-    <string name="content_description_leave_the_group">Lahku grupist</string>
-    <string name="content_description_delete_the_group">Kustuta grupp</string>
+    <string name="content_description_leave_the_conversation">Lahku grupist</string>
+    <string name="content_description_delete_the_conversation">Kustuta grupp</string>
```

Comment:
if the English copy was updated i think any translation should be deleted this way it defaults back to the correct eng one